local-cli/__mocks__/chalk.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
I think I got the reason why it fails to merge.
We have another chalk.js mock: react-native-github/packager/react-packager/src/Activity/__mocks__/chalk.js.

@GantMan, could you merge them into one?